src/main/views/error.njk

```diff
@@ -1,5 +1,15 @@
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
 {% extends "template.njk" %}
 
+{% block beforeContent %}
+  {{ govukBackLink({
+    text: text,
+    href: '#',
+    attributes: {
+      onClick: "history.back();"
```

Comment:
This unfortunately won't work on Safari.

There's another PR (739) which has the fix for this.

Reply:
Fixed.